### PR TITLE
[UK] Getting Started (excluding FAQ)

### PR DESCRIPTION
**Pull request by atimonder1**
- [x] Help Centre
- [x] Help Centre (Account Restrictions)
- [x] Installation
- [x] Installation (macOS)
- [x] Registration
- [x] Welcome

### CODE REVIEW

**Comment by atimonder1** [comment]
I'm a little dumb in terms of translating the FAQ, I just don't understand how to choose the correct words lmao. PR is ready for review

**Review by sr229 — CHANGES_REQUESTED** [review]
We already established that `en.md` is British English so you might want to merge your changes there.

**Comment by MegaApplePi** [comment]
@sr229 UK is Ukrainian.

File: wiki/Help_Centre/uk.md [hunk 1]
```diff
@@ -0,0 +1,964 @@
...
+
+#### Я отримую помилку про "стиль шрифту", коли я хочу відкрити гру!
+
+**Ви видалили або втратили важливі системні шрифти, які були інстальовані з базовим встановленням Windows. Скоріше всього, це не єдина проблема з якою ви стикнетесь!**
+
+Якщо ви бачите цю помилку під час запуску osu! або це апдейтер, це значить, що ви видалили або в іншому випадку втратили важливі системні шрифти, котрі йдуть разом з Windows. Ці шрифти використовуються по всій Windows OS, так що ви можете побачити і інші проблеми, як неправильні шрифти, відсутній текст, і інші збої програм в Windows.
+
+Ось колекція звичайних шрифтів, яка використовуються в різних версіях Windows і в Інтернеті, які мають допомогти:
+
+- [Microsoft core web fonts](https://web.archive.org/web/20020124085641/http://www.microsoft.com/typography/fontpack/default.htm) (особливо Arial, Times New Roman, Trebuchet MS, and Verdana)
```
Comment (TicClick) [inline wiki/Help_Centre/uk.md:264]:
```suggestion
- [Microsoft core web fonts](https://web.archive.org/web/20020124085641/http://www.microsoft.com/typography/fontpack/default.htm) (особливо Arial, Times New Roman, Trebuchet MS i Verdana)
```

File: wiki/Help_Centre/uk.md [hunk 2]
```diff
@@ -0,0 +1,964 @@
...
+- [Tahoma](https://freefontsfamily.com/tahoma-font-free)
+- [Windows Live Essentials](https://support.microsoft.com/en-us/help/2434419/windows-live-essentials-2011) (Segoe UI)
+- [Microsoft JhengHei](https://microsoft.com/en-us/download/details.aspx?&id=12072) (微軟正黑體) (Традиційний Chinese UI шрифт)
```
Comment (TicClick) [inline wiki/Help_Centre/uk.md:267]: _Традиційний Chinese UI шрифт_ — это шрифт для традиционного китайского, переведи фразу целиком

File: wiki/Help_Centre/uk.md [hunk 3]
```diff
@@ -0,0 +1,964 @@
...
+
+Якщо ви зустрічаєте якісь проблеми з цим процесом, будь ласка, відправте нам повідомлення на нашу поштову скриньку [store@ppy.sh](mailto:store@ppy.sh) і ми з радістю допоможем вам.
+
+---
+
+## Вирішення проблем
+
+### Часті питання
+
+#### Що за 'log files' і як я можу зробити їх?
```
Comment (TicClick) [inline wiki/Help_Centre/uk.md:923]: log files нужно перевести

File: wiki/Help_Centre/uk.md [hunk 4]
```diff
@@ -0,0 +1,964 @@
...
+
+**Логові файли є детальним записом того, що ваш ігровий клієнт робить в якийсь проміжок часу. Ми використовуємо його для того, щоб роспізнати, що виникає цю проблему.**
+
+Ці файли дуже зручні і можуть зробити вирішення навіть дуже тяжких тривіальних проблем.
+
+Якщо член команди підтримки запитав вас на наявність цих логів, ось як дістати їх:
+
+1. Відкрийте *osu*.
```
Comment (TicClick) [inline wiki/Help_Centre/uk.md:931]:
```suggestion
1. Відкрийте osu!.
```

**Review by TicClick — CHANGES_REQUESTED** [review]
I don't know Ukrainian, but these are my findings so far. I'll also see if I can find someone proficient to give it a read.

**Comment by TicClick** [comment]
(English for clarity)

I've asked a native speaker to proof-read this, and the feedback was less than satisfactory (not a quote):
- half-assed, sound more like a machine translation;
- would take quite some time to get it into appropriate shape;
- highly recommended to use Russian articles as a basis for translation, as they share most of grammar rules.

make of that what you will, with that I don't feel the pull request is good enough. you probably want to wait for a guy himself to show up either with improved translation, or with a new review, at least he claimed he'd do so.

**Pull request closed without merging**